internal/commands/pull.go

```diff
@@ -52,5 +91,15 @@ func runPull(dockerCli command.Cli, name string) error {
 	}
 	fmt.Fprintf(os.Stdout, "Successfully pulled %q (%s) from %s\n", bndl.Name, bndl.Version, ref.String())
 
+	if opts.serviceImages {
+		ctx := context.Background()
+		for name, image := range bndl.Images {
```

Comment:
I wonder if it makes sense to parallelize pulls here 🤔

Reply:
Let me know what you all think. I will warn you that I know just enough about Golang that I might be dangerous doing this bit :)